front/src/common/api/osrdEditoastApi.ts

```diff
@@ -1171,7 +1171,25 @@ export type PostStdcmApiResponse =
       error?: string;
     };
 export type PostStdcmApiArg = {
-  stdcmRequest: StdcmRequest;
+  body: {
+    comfort: Comfort;
+    end_time?: number;
+    infra_id: number;
+    margin_after?: number;
+    margin_before?: number;
+    maximum_departure_delay?: number;
+    maximum_run_time?: number;
+    rolling_stock_id: number;
+    rolling_stocks: number[];
+    speed_limit_tags?: string;
+    standard_allowance: AllowanceValue;
+    start_time?: number;
+    steps: {
+      duration?: number;
+      waypoints?: Waypoint;
```

Comment:
I think duration and waypoints are required
```suggestion
      duration: number;
      waypoints: Waypoint;
```